src/rrdp.rs

```diff
@@ -680,15 +688,48 @@ impl CurrentObject {
         &self.data
     }
 
-    pub fn hash(&self) -> Hash {
-        Hash::from_data(&self.data)
+    pub fn since(&self) -> Time {
+        self.since
+    }
+
+    /// Fixes the since time based on the actual parsed object.
+    /// Returns an error if the object cannot be parsed.
```

Comment:
Is it worth adding a comment here explaining _why_ we do this and in what way it "fixes" the time?

Reply:
add a comment where fn fix_since is called.